### PR TITLE
Implementing Hours Screen

### PR DESCRIPTION
**Pull request by griffisr**
Hard coded for the time being

### CODE REVIEW

**Review by mdomas1 — DISMISSED** [review]

File: src/screens/HoursScreen.tsx [hunk 1]
```diff
@@ -1,17 +1,101 @@
...
+     'Mon                                 5:00pm-1:00am',
+     'Tue                                 5:00pm-1:00am',
+     'Wed                                 5:00pm-1:00am',
+     'Thu                                 5:00pm-1:00am',
+     'Fri                                 5:00pm-1:00am',
+     'Sat                                 5:00pm-1:00am',]
+  }
+];
+
+const Item = ({title}) => (
```
Comment (nathan815) [inline src/screens/HoursScreen.tsx:43]: To fix the props validation error add a TypeScript type to this component's props like this:

```
const Item = ({ title }: { title: string }) => (
  <View style={styles.item}>
    <Text style={styles.title}>{title}</Text>
  </View>
);
```
Or can also be written as:

```
const Item = (props: { title: string }) => (
  <View style={styles.item}>
    <Text style={styles.title}>{props.title}</Text>
  </View>
);
```

That should get you past this error so you can commit the linted code.

File: src/screens/HoursScreen.tsx [hunk 2]
```diff
@@ -1,17 +1,80 @@
...
+  },
+  {
+    title: 'Request SeeUs Van',
+    data: [],
+  },
+  {
+    title: 'Request A Jump/Unlock',
+    data: [],
+  },
+];
```
Comment (nathan815) [inline src/screens/HoursScreen.tsx:25]: No hours data?

Comment (griffisr) [reply src/screens/HoursScreen.tsx:25]: Smh I deleted it trying to fix commit and forgot to add it back

**Review by nathan815 — APPROVED** [review]

**Comment by nathan815** [comment]
Good to merge this?

**Pull request merged**